storage-resize-images/extension.yaml

```diff
@@ -114,6 +114,19 @@ params:
       Invalid sizes, must be a comma-separated list of WIDTHxHEIGHT values.
     required: true
 
+  - param: KEEP_FILE_METADATA
+      type: select
+      label: Keep Original Custom Metadata? (Yes/No)
+      description: >-
+        Copy the custom metadata from the original image to the resized image?
```

Comment:
It may be best to imply there may not be any existing metadata? cc @rachelsaunders 

> Copy any custom metadata from the original image to the resized image?